src/main/java/com/google/sps/servlets/AnalyzedThread.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.util.Random;
+
+public class AnalyzedThread {
+  public final static ImmutableList<String> titlesList =
+      ImmutableList.of("Where is this supposed to go?", "This program needs to work better",
+          "The wifi needs to be more consistent", "This could be better",
+          "Maybe this could be structured a different way");
+  public final static ImmutableList<Double> sentimentList =
+      ImmutableList.of(-.3, -.5, -.1, -.2, -.5);
+  public final static ImmutableList<Integer> upvotesList = ImmutableList.of(30, 50, 10, 5, 12);
+  public final static ImmutableList<String> urlList =
+      ImmutableList.of("http://www.youtube.com", "http://www.reddit.com", "http://www.facebook.com",
+          "http://www.yahoo.com", "http://www.amazon.com");
+  private final String threadTitle;
+  private final double sentiment;
+  private final int upvotes;
+  private final int timestamp;
+  private final String threadUrl;
+  private final static Random rand = new Random();
```

Comment:
nit: usually, it's nice to order the field variable as is:
 1. final static variables
 2. static (non-final) variables
 3. final variables
 4. mutable variables